src/main/java/org/lantern/Launcher.java

```diff
@@ -279,26 +279,11 @@ void launch() {
             }
         }
         
-        proxyTracker = instance(ProxyTracker.class);
+        //proxyTracker = instance(ProxyTracker.class);
         httpClientFactory = instance(HttpClientFactory.class);
 
         s3ConfigFetcher = instance(S3ConfigFetcher.class);
         
-        if (checkFallbacks) {
```

Comment:
With these changes, we don't need to handle the check-fallbacks mode at all anymore, right? There's other places in this class we can remove references to it, and we also can remove support for --check-fallbacks as a command line option.